site/content/docs/user/known-issues.md

```diff
@@ -217,3 +218,17 @@ This problem seems to be related to a [bug in Docker][moby#9939].
 [moby#9939]: https://github.com/moby/moby/issues/9939
 [Docker resource lims]: https://docs.docker.com/docker-for-mac/#advanced
 [snap]: https://snapcraft.io/
+
+## Failures involving mistmatched kubectl versions
```

Comment:
nit:  I guess that is better if you add this new section before the links section, it will be easy to read in the future

Reply:
Thank you for the feedback, will do